listenbrainz_spark/stats/sitewide/recording.py

```diff
@@ -17,33 +18,50 @@ def get_recordings(table: str, limit: int = SITEWIDE_STATS_ENTITY_LIMIT):
     # order of collected results is not guaranteed so sort again
     # with sort_array.
     result = run_query(f"""
-        WITH intermediate_table as (
-            SELECT first(recording_name) AS any_recording_name
+        WITH user_counts as (
+            SELECT user_name
+                 , first(recording_name) AS recording_name
                  , recording_mbid
-                 , first(artist_name) AS any_artist_name
+                 , first(artist_name) AS artist_name
                  , artist_credit_mbids
-                 , nullif(first(release_name), '') as any_release_name
+                 , nullif(first(release_name), '') as release_name
                  , release_mbid
-                 , count(*) as listen_count
+                 , LEAST(count(*), {user_listen_count_limit}) as listen_count
```

Comment:
This causes the final listen count to contain the capped count not the actual listen count. An alternative is to do both counts, the actual listen count and the capped count. Sort on the capped count and assign rank like 1, 2, 3 based on it and have the actual listen count field available separetely for anyone who's interested in it.

Reply:
Why should we care about the actual listen count here? I think as it is it sounds fine to me.

Reply:
I was thinking some one might be interested in the actual listen counts. I am fine either way, will merge as is then for now.